modules/basic/config.nix

```diff
@@ -140,7 +143,7 @@ in {
       ''}
       ${optionalString cfg.spellChecking.enable ''
         set spell
-        set spelllang=${toString cfg.spellChecking.language}
+        set spelllang=${builtins.concatStringsSep "," cfg.spellChecking.languages}${optionalString cfg.spellChecking.enableProgrammingWordList ",programming"}
```

Comment:
would be nice to have spaces after commas for a little added readability, `", "` instead of `","`

Reply:
Idk if that's supported

Reply:
Yeah I'm pretty sure it's not